ddtrace/_trace/_span_pointer.py

```diff
@@ -17,6 +18,18 @@ class _SpanPointerDirection(Enum):
     DOWNSTREAM = "d"
 
 
+class _SpanPointerDescription(NamedTuple):
```

Comment:
Do we use this anywhere? Maybe I don't get it.

Reply:
not yet, but we will in https://github.com/DataDog/dd-trace-py/pull/10689

Reply:
Ah I checked one ahead, but not multiple ahead, thanks.